docs/developer.md

```diff
@@ -1,20 +1,13 @@
-# blob-uploader.js
+# bm-uploader.js
 
-This library wraps the blob uploader service [Blob uploader](https://github.com/blinkmobile/blob-uploader) and AWS S3 to allow uploading and retrieval of blobs from S3 without having to work with AWS credentials and the AWS SDK directly.
+This library wraps the blob uploader service [Blob uploader](https://github.com/blinkmobile/blob-uploader) and AWS S3 to allow uploading and retrieval of content from S3 without having to work with AWS credentials and the AWS SDK directly.
 
 ## Usage
-1.  Initialise the BlobUploader with the URL of the blob uploader service, e.g. 
+1.  Initialise the bmUploader with the URL of the blob uploader service, e.g. 
 ```
-new BlobUploader('https://bm-blob-uploader-dev.api.blinkm.io/')
+new bmUploader('https://bm-blob-uploader-dev.api.blinkm.io/')
```

Comment:
These examples imply `uploadContent()` and `retrieveContentUrl()` is a static method on the globally available `bmUploader` object.

Is there a reason we went away from `BMUploader` or `BmUploader`?